sdk/app_cpu1/common/drv/timing_manager.h

```diff
@@ -13,7 +13,8 @@
 #define ISR0_PRIORITY            (0x08)
 #define ISR_RISING_EDGE          (0x3)
 #define DEFAULT_SENSOR_ENABLE    (0x00)
-#define LOWER_16_MASK            (0x00FF)
+#define DEFAULT_PWM_RATIO        (0x1)
```

Comment:
I see this value is replacing what was previous `20`, but `0x1` does not equal `20`? Is this correct?

Reply:
yes this is correct, i had 20 when i was testing and didn't want the interrupt going too fast, i've now just made the default 1 (so if the PWM trigger is both, it will be every high/low rather than every 20 high/low)

Reply:
Ah I see.

Once #377 is implemented, will there be some code elsewhere which sets the PWM ratio such that we achieve the desired 100 kHz PWM and 10 kHz interrupt operation? Or will we change this `DEFAULT_PWM_RATIO` define to do this?

Reply:
I'm not 100% sure yet, I'll get working on that issue and figure out the best way to do this